Competition2017/src/competition/subsystems/climbing/commands/ClimbClimbingCommand.java

```diff
@@ -0,0 +1,33 @@
+package competition.subsystems.climbing.commands;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.climbing.ClimbingSubsystem;
+
+import xbot.common.command.BaseCommand;
+
+public class ClimbClimbingCommand extends BaseCommand {
```

Comment:
I think we can improve this name. What about incorporating the term `ascend` (here and in other files)?